package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cvr-view",
-  "version": "1.1.0",
+  "version": "1.0.0",
```

Comment:
It is like a countdown for the space shuttle. Actually, there was never a 1.0 release but I got excited one day and made it 1.1. It should stay at 1.0 until it actually gets a tag and release on npm, which I hope will be pretty soon.